### PR TITLE
Integrate updates css

### PR DESCRIPTION
**Pull request by roshkins**
Some of the css refers to features that needs to be implemented.

### CODE REVIEW

File: lib/css/modules/_nightMode.scss [hunk 1]
```diff
@@ -1382,3 +1382,19 @@
 		}
 	}
 }
+
+
+.res-nightmode:not(.res-nightMode-coloredLinks){
```
Comment (andytuba) [inline lib/css/modules/_nightMode.scss]: Does nightmode options include a colored links boolean body class option yet? Should probably be enabled by default.

File: lib/css/res.scss [hunk 2]
```diff
@@ -43,6 +43,9 @@
 
 body.res-console-open {
 	overflow: hidden;
+	&.front-page.res-hide-tagline-frontpage .tagline {
```
Comment (andytuba) [inline lib/css/res.scss]: Looks like a merge conflict resolution issue here, this probably shouldn't be inside the console open block.

File: lib/css/modules/_newCommentCount.scss [hunk 3]
```diff
@@ -3,6 +3,10 @@
 	color: orangered;
 }
 
+.new-comment .usertext-body {
```
Comment (andytuba) [inline lib/css/modules/_newCommentCount.scss]: This seems like it should move inside a "new comments count" block.

File: lib/css/modules/_neverEndingReddit.scss [hunk 4]
```diff
@@ -115,6 +115,10 @@
 #NREPause {
 	cursor: pointer;
 
+	.res-neverEndingReddit-hideFloatingPauseButton & {
```
Comment (andytuba) [inline lib/css/modules/_neverEndingReddit.scss]: This should become a disabled boolean body class option, with a migration to enable it if found in a stylesheet module's option  body class value,  and filter out the row containing it.

Comment (andytuba) [inline lib/css/modules/_neverEndingReddit.scss]: *there should be added

**Comment by erikdesjardins** [comment]
Thanks!

<sup>(so the build passed but it's marked as errored...hmmm)</sup>

**Pull request merged**